lightkurve/periodogram.py

```diff
@@ -697,23 +710,34 @@ def from_lightcurve(lc, min_frequency=None, max_frequency=None,
                         "These are not evenly spaced in frequency space.\n"
                         "Method has been set to 'slow' to allow for this.")
 
+        flux_scaling = 1e6 if normalization == 'psd' else 1.
         if float(astropy.__version__[0]) >= 3:
-            LS = LombScargle(time, lc.flux * 1e6,
+            LS = LombScargle(time, lc.flux * flux_scaling,
                              nterms=nterms, normalization='psd', **kwargs)
             power = LS.power(frequency, method=method)
         else:
-            LS = LombScargle(time, lc.flux * 1e6,
+            LS = LombScargle(time, lc.flux * flux_scaling,
                              nterms=nterms, **kwargs)
             power = LS.power(frequency, method=method, normalization='psd')
 
-        # Normalise the according to Parseval's theorem
-        norm = np.std(lc.flux * 1e6)**2 / np.sum(power)
-        power *= norm
+        # Power spectral density
+        if normalization == 'psd':
+            # Normalise according to Parseval's theorem
+            norm = np.std(lc.flux * 1e6)**2 / np.sum(power)
+            power *= norm
+
+            power = power * (cds.ppm**2)
+
+            # Rescale power to units of ppm^2 / [frequency unit]
+            power = power / fs
 
-        power = power * (cds.ppm**2)
+        # Amplitude spectrum
+        elif normalization == 'amplitude':
+            factor = np.sqrt(4./len(lc.time))
+            power = np.sqrt(power) * factor * 1e6
```

Comment:
Above you create `flux_scaling` and then set it to `1`. However, here you multiply by 1e6. Why is this not the `flux_scaling` from above, set to `1e6`?